### ISSUE TITLE
If Template component is deactivated - Widget Select Template Field turns to a simple form field 

### ISSUE THREAD

**Issue by quasel** [pods-framework/pods#3028]
If Template component is deactivated - Widget Select Template Field turns to a simple form field that obviously doesnt work

![screenshot_2015-06-11_22 43 47](https://cloud.githubusercontent.com/assets/9464217/8124228/7f365020-10d8-11e5-85d0-8ec28023b7ce.png)

Deactivate / remove it whatever

**Comment by sc0ttkclark** [comment]
Technically this should be a textarea with magic tags accepted for basic templating syntax (magic tags only).

**Comment by jimtrue** [comment]
Right, it should not show the 'template' line and ONLY show the Custom Template box since that's the only option with Pods Templates.

**Comment by pglewis** [comment]
There is already a dedicated "Custom Template" textarea field in the list items widget.  

If you have templates enabled and have available templates, the "Template" field is a drop down that includes "Custom Template" as an option.  Keeping it as a drop-down with "Custom Template" as the lone option makes the most sense to me if templates are disabled or there are no templates.

**Comment by quasel** [comment]
Sry bad Screenshot @sc0ttkclark 
@pglewis sound's like a good and easy to understand way 
Other Option just grey it out ;) btw what happens if you choose a template and fill out the Custom Template Field? didn't check
